test/test_get_a_specific_question.py

```diff
@@ -0,0 +1,14 @@
+from .test_base import *
```

Comment:
This is a bad import especially when you have many imports. Its bad to import everything from a file. If a module has similar names to modules imported from other files, it will overwrite them